### ISSUE TITLE
Failed to run with "Object 0xX is not an instance of type generic-pc-machine" message

### ISSUE THREAD

**Issue by jpirko** [arighi/virtme-ng#87]
$ /home/x/virtme-ng/vng -v
virtme: waiting for virtiofsd to start
virtme: use 'microvm' QEMU architecture
/builddir/build/BUILD/qemu-8.1.3/include/hw/i386/pc.h:135:PC_MACHINE: Object 0x55af13219130 is not an instance of type generic-pc-machine

Running on Fedora 39, I'm getting this issues. I'm getting the same with hand-build qemu (HEAD 8f3f329f5e0117bd1a23a79ab751f8a7d3471e4b). On Fedora 38, I'm pretty sure didn't see this, not sure why though.

**Comment by arighi** [comment]
I'm not seeing this with qemu 8.2.1, I'll do some tests with 8.1.3 and the latest qemu from git.

Does it work if you run `vng --disable-microvm`?

**Comment by jpirko** [comment]
yeah, with --disable-microvm, qemu runs and I don't see this issue.
